charts/scalardl-auditor-monitoring/values.yaml

```diff
@@ -392,6 +290,9 @@ prometheus:
           namespaces:
             own_namespace: true
 
+  extraScrapeConfigs: |
+    {{ include "scalardl-auditor-monitoring.prometheusExtraScrapeConfig" . }}
```

Comment:
The scrape configuration for Envoy is set here, `prometheus.extraScrapeConfigs`, if we set `global.envoy.enabled=true`.